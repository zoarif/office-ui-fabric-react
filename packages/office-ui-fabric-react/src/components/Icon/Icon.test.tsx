import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { Icon } from './index';

describe('Icon', () => {
  it('renders Icon correctly', () => {
    const component = renderer.create(<Icon iconName='CompassNW' />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});