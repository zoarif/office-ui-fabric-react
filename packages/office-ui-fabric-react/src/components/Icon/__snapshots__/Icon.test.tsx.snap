// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon renders Icon correctly 1`] = `
<i
  aria-hidden={true}
  className=

      {
        display: inline-block;
      }
  data-icon-name="CompassNW"
  role="presentation"
/>
`;
